browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -428,4 +448,133 @@ void AIChatUIPageHandler::OnGetPremiumStatus(
   }
 }
 
+#if BUILDFLAG(ENABLE_PRINT_PREVIEW)
+void AIChatUIPageHandler::BitmapConverterDisconnected() {
+  DLOG(ERROR) << __func__;
+  PreviewCleanup();
+}
+
+void AIChatUIPageHandler::OnGetBitmaps(
+    const std::optional<std::vector<SkBitmap>>& bitmaps) {
+  VLOG(3) << __func__ << ": bitmap size: " << (bitmaps ? bitmaps->size() : -1);
```

Comment:
Do you manipulate with `bitmaps` on UI thread?
If `bitmaps` is huge, even simple operations could result in a short UI hang.

Reply:
Do you mean any huge vectors in general or a huge vector with SkiBitmap specifically? 
I can remove this VLOG though.

Reply:
Also these bitmaps will be OCR on different thread in `PreviewPageTextExtractor`